content/blogs/YYYY-MM-DD-CI-CD-kestra-comparison.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Kestra vs. Popular CI/CD Tools: When to Choose an Orchestration Solution"
+description: "Learn when to to choose an orchestrator rather than a CI/CD solution"
+date: xxxxxxx
+category: Solutions
+author:
+  name: Federico Trotta
+  image: "ftrotta"
+image: /blogs/xxxxx.jpg
+---
+
+In a recent [blog](./2024-09-18-what-is-an-orchestrator.md), we've defined what an orchestrator is, what are the differences between orchestration and automation, and how orchestration can help you to automate your workflows.
+
+Now, it's time to answer the question: when should you choose an orchestrator over a CI/CD solution?
+
+In this article, we'll compare Kestra to popular CI/CD solutions to help you get a practical understanding of when to choose an orchestrator over a CI/CD tool. In particular, at the end of the article, we'll provide a deep comparison between Kestra and Jenkins.
+
+Ready? Let's dive in!
+
+## Comparing Kestra to CI/CD Tools
+As we've already clarified the difference between orchestration and automation in the above-mentioned blog, let’s directly dive into how Kestra stacks up against popular CI/CD tools. 
+
+### Kestra Vs. GitHub Actions

```

Comment:
The main thing that I would point out for GH Actions is that to test if your Action is setup correctly, you need to commit it to your GitHub repository which can be annoying and slow, as well as mess up your commit history.

Kestra makes that easier with the built in editor where you can tweak something and execute straight away. Can you add something about this too?